model/clusters_mgmt/v1/flavour_type.model

```diff
@@ -19,13 +19,18 @@ limitations under the License.
 class Flavour {
 	// Default _Amazon Web Services_ settings of the cluster.
 	//
-	// These can be overriden specifying in the clsuter itself a different set of settings.
+	// These can be overridden specifying in the cluster itself a different set of settings.
 	AWS AWSFlavour
 
+	// Default _Google Cloud Platform_ settings of the cluster.
+	//
+	// These can be overridden specifying in the cluster itself a different set of settings.
+	GCP GCPFlavour
+
 	// Number of nodes that will be used by default when creating a cluster that uses
 	// this flavour.
 	//
-	// These can be overriden specifying in the cluster itself a different number of nodes.
+	// These can be overridden specifying in the cluster itself a different number of nodes.
```

Comment:
Again, only the compute nodes can be overridden

Reply:
Thanks. I'll remove this comment.